Perform corrections to Line Analysis Flux Results (#1564)
spacetelescope/jdaviz: docs/specviz/plugins.rst

The statistics returned include the line centroid, gaussian sigma width, gaussian FWHM,
total flux, and equivalent width.

The line flux of JWST data are automatically converted to Watts/meter^2 and multiplied by

> PIXAR_SR if availble.

Reviewer: ```suggestion
``PIXAR_SR`` if available.
```

The statistics returned include the line centroid, gaussian sigma width, gaussian FWHM,
total flux, and equivalent width.

> The line flux results are automatically converted to Watts/meter^2 if possible.

Reviewer 2: ```suggestion
The line flux results are automatically converted to Watts/meter^2, when appropriate.
```
